package.json

```diff
@@ -70,6 +70,7 @@
     "eslint-plugin-promise": "^5.1.1",
     "eslint-plugin-react": "^7.27.0",
     "eslint-plugin-react-hooks": "^4.3.0",
+    "identity-obj-proxy": "^3.0.0",
```

Comment:
@vidyanambiar would you mind explaining what this new dependency is for? Thanks!

Reply:
@florkbr This dependency helps us mock css modules for Jest using an ES6 proxy. I got this idea from Filip's PR for unit testing the list view (#116). Without this change, I noticed the HorizontalNav tests failing on an error in the `backdrop.css` file within patternfly react-styles. 
```
 FAIL   lib-utils  packages/lib-utils/src/components/horizontal-nav/HorizontalNav.test.tsx
  ● Test suite failed to run

    Jest encountered an unexpected token

    Jest failed to parse a file. This happens e.g. when your code or its dependencies use non-standard JavaScript syntax, or when Jest is not configured to support such syntax.

    Out of the box Jest supports Babel, which will be used to transform your files into valid JS based on your Babel configuration.

    By default "node_modules" folder is ignored by transformers.

    Here's what you can do:
     • If you are trying to use ECMAScript Modules, see https://jestjs.io/docs/ecmascript-modules for how to enable it.
     • If you are trying to use TypeScript, see https://jestjs.io/docs/getting-started#using-typescript
     • To have some of your "node_modules" files transformed, you can specify a custom "transformIgnorePatterns" in your config.
     • If you need a custom transformation specify a "transform" option in your config.
     • If you simply want to mock your non-JS modules (e.g. binary assets) you can stub them out with the "moduleNameMapper" config option.

    You'll find more details and examples of these config options in the docs:
    https://jestjs.io/docs/configuration
    For information about custom transformations, see:
    https://jestjs.io/docs/code-transformation

    Details:

    /Users/vidyanambiar/Documents/hac-stuff/dynamic-plugin-sdk/node_modules/@patternfly/react-styles/css/components/Backdrop/backdrop.css:1
    ({"Object.<anonymous>":function(module,exports,require,__dirname,__filename,jest){.pf-c-backdrop {
                                                                                      ^

    SyntaxError: Unexpected token '.'

      at Runtime.createScriptFromCode (../../node_modules/jest-runtime/build/index.js:1728:14)
      at Object.<anonymous> (../../node_modules/@patternfly/react-styles/css/components/Backdrop/backdrop.js:3:1)
```

Reply:
@vidyanambiar thanks for explaining.

For now this change seems fine. We may want to consider _correctly_ loading these CSS files in the future (versus using `identity-obj-proxy`) in our unit tests. I'm mainly thinking if we ever introduce conditional style logic or if we'd like to protect ourselves from broken style sheets on upgrades in the future (would our build/tests catch it?). We don't necessarily want to test Patternfly's styles however, nor would we really do that from a unit test perspective - perhaps visual snapshot testing with a tool such as https://loki.js.org/.